### PR TITLE
feat(ui/search-xref): add xref search UI

### PR DESCRIPTION
**Pull request by sidvishnoi**
[Demo at respec-preview](https://respec-preview.netlify.com/?spec=https%3A%2F%2Fw3c.github.io%2Fpayment-request%2F&version=https%3A%2F%2Fdeploy-preview-2453--respec-pr.netlify.com%2Frespec-w3c.js)

### CODE REVIEW

File: assets/ui.css [hunk 1]
```diff
@@ -401,3 +401,18 @@
...
+  min-height: 550px;
+  height: 100%;
+  overflow: hidden;
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+
+#xref-ui:not(.ready) {
+  background: url("http://loadinggif.com/images/image-selection/8.gif")
```
Comment (sidvishnoi) [inline assets/ui.css]: Where can we host it?

Comment (marcoscaceres) [reply assets/ui.css]: can we inline it?

Comment (marcoscaceres) [reply assets/ui.css]: (also, is there an SVG version?)

Comment (sidvishnoi) [reply assets/ui.css]: Inline: Can do depending on size. This gif is 3KB.
I liked this gif 😺 Will see if I can find a similar SVG

Comment (marcoscaceres) [reply assets/ui.css]: The gif is nice. We can just host it on ReSpec.org?

File: src/ui/search-xref.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    title: "Search definitions",
+  },
+};
+const lang = defaultLang in localizationStrings ? defaultLang : "en";
+const l10n = localizationStrings[lang];
+
+const button = ui.addCommand(l10n.title, show, "Ctrl+Shift+Alt+x", "📚");
+
+function show() {
+  const iframe = hyperHTML`
```
Comment (marcoscaceres) [inline src/ui/search-xref.js]: We are not really doing any fancy rendering here... so maybe we should avoid using hyper?

**Pull request merged**